logger/zap_logger.go

```diff
@@ -84,14 +73,15 @@ var (
 )
 
 type Configuration struct {
-	Level    string
-	Encoding string
-
+	Level         string
+	Encoding      string
 	StdoutEnabled bool
-
-	SyslogEnabled  bool
-	SyslogFacility int
-	SyslogTag      string
+	SyslogEnabled bool
+	// SyslogFacility is one of `KERN,USER,MAIL,DAEMON,AUTH,SYSLOG,LPR,NEWS,UUCP,CRON,AUTHPRIV,FTP,LOCAL0,
+	// LOCAL1,LOCAL2,LOCAL3,LOCAL4,LOCAL5,LOCAL6,LOCAL7`
+	SyslogFacility string
```

Comment:
Breaking change: previously we could only set integers from 0 to 7 that get translated to `LOCAL<N>`.

While this is handy, it's not really generic enough. 

I think setting `LOCAL<N>` by ourselves is not that big of a problem